### ISSUE TITLE
The core tests are failing to build on Windows

### ISSUE THREAD

**Issue by murgatroid99** [grpc/grpc#11312]
https://grpc-testing.appspot.com/job/gRPC_master_windows/2665/testReport/junit/(root)/aggregate_tests/run_tests_c_windows_dbg_native/

```
"C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\buildtests_c.sln" (default target) (1) ->

       "C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\vcxproj\test\alarm_test\alarm_test.vcxproj.metaproj" (default target) (2) ->

       "C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\vcxproj\.\grpc_test_util\grpc_test_util.vcxproj.metaproj" (default target) (3) ->

       "C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\vcxproj\.\grpc\grpc.vcxproj.metaproj" (default target) (5) ->

       "C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\vcxproj\grpc\grpc.vcxproj" (default target) (161) ->

         C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\\..\src/core/ext/census/intrusive_hash_map.h(87): error C2054: expected '(' to follow 'inline' [C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\vcxproj\grpc\grpc.vcxproj]

         C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\\..\src/core/ext/census/intrusive_hash_map.h(87): error C2085: 'hm_index_compare' : not in formal parameter list [C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\vcxproj\grpc\grpc.vcxproj]

         C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\\..\src/core/ext/census/intrusive_hash_map.h(87): error C2143: syntax error : missing ';' before '{' [C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\vcxproj\grpc\grpc.vcxproj]

         C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\\..\src\core\ext\census\intrusive_hash_map.c(23): error C2370: 'inline' : redefinition; different storage class [C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\vcxproj\grpc\grpc.vcxproj]

         C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\\..\src\core\ext\census\intrusive_hash_map.c(23): error C2054: expected '(' to follow 'inline' [C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\vcxproj\grpc\grpc.vcxproj]

         C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\\..\src\core\ext\census\intrusive_hash_map.c(23): error C2085: 'chunked_vector_hasher' : not in formal parameter list [C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\vcxproj\grpc\grpc.vcxproj]

         C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\\..\src\core\ext\census\intrusive_hash_map.c(23): error C2143: syntax error : missing ';' before '{' [C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\vcxproj\grpc\grpc.vcxproj]

         C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\\..\src\core\ext\census\intrusive_hash_map.c(31): error C2370: 'inline' : redefinition; different storage class [C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\vcxproj\grpc\grpc.vcxproj]

         C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\\..\src\core\ext\census\intrusive_hash_map.c(31): error C2054: expected '(' to follow 'inline' [C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\vcxproj\grpc\grpc.vcxproj]

         C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\\..\src\core\ext\census\intrusive_hash_map.c(32): error C2085: 'get_mutable_bucket' : not in formal parameter list [C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\vcxproj\grpc\grpc.vcxproj]

         C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\\..\src\core\ext\census\intrusive_hash_map.c(32): error C2143: syntax error : missing ';' before '{' [C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\vcxproj\grpc\grpc.vcxproj]

         C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\\..\src\core\ext\census\intrusive_hash_map.c(40): error C2370: 'inline' : redefinition; different storage class [C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\vcxproj\grpc\grpc.vcxproj]

         C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\\..\src\core\ext\census\intrusive_hash_map.c(40): error C2054: expected '(' to follow 'inline' [C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\vcxproj\grpc\grpc.vcxproj]

         C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\\..\src\core\ext\census\intrusive_hash_map.c(40): error C2085: 'get_bucket' : not in formal parameter list [C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\vcxproj\grpc\grpc.vcxproj]

         C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\\..\src\core\ext\census\intrusive_hash_map.c(40): error C2143: syntax error : missing ';' before '{' [C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\vcxproj\grpc\grpc.vcxproj]

         C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\\..\src\core\ext\census\intrusive_hash_map.c(49): error C2370: 'inline' : redefinition; different storage class [C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\vcxproj\grpc\grpc.vcxproj]

         C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\\..\src\core\ext\census\intrusive_hash_map.c(49): error C2054: expected '(' to follow 'inline' [C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\vcxproj\grpc\grpc.vcxproj]

         C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\\..\src\core\ext\census\intrusive_hash_map.c(49): error C2085: 'RestSize' : not in formal parameter list [C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\vcxproj\grpc\grpc.vcxproj]

         C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\\..\src\core\ext\census\intrusive_hash_map.c(49): error C2143: syntax error : missing ';' before '{' [C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\vcxproj\grpc\grpc.vcxproj]

         C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\\..\src\core\ext\census\intrusive_hash_map.c(208): error C2370: 'inline' : redefinition; different storage class [C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\vcxproj\grpc\grpc.vcxproj]

         C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\\..\src\core\ext\census\intrusive_hash_map.c(208): error C2054: expected '(' to follow 'inline' [C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\vcxproj\grpc\grpc.vcxproj]

         C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\\..\src\core\ext\census\intrusive_hash_map.c(210): error C2085: 'intrusive_hash_map_internal_insert' : not in formal parameter list [C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\vcxproj\grpc\grpc.vcxproj]

         C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\\..\src\core\ext\census\intrusive_hash_map.c(210): error C2143: syntax error : missing ';' before '{' [C:\jenkins\workspace\gRPC_master_windows\workspace_c_windows_dbg_native\vsprojects\vcxproj\grpc\grpc.vcxproj]
```

**Comment by makdharma** [comment]
Fixed with #11337

**Comment by murgatroid99** [comment]
This is still happening on 1.4.x
